Documentation/how-to-create-an-erd.md

```diff
@@ -0,0 +1,17 @@
+# How To create an entity relation diagram
+
```

Comment:
Consider making a small introduction to what it is and why we do it. Maybe consider including a small screenshot.